### PR TITLE
vcrun2022: update checksum

### PR DESCRIPTION
**Pull request by Gcenx**

### CODE REVIEW

**Comment by austin987** [comment]
You also need to update the x86 version.

**Comment by Gcenx** [comment]
I’ll retest but didn’t see any complains about the x86 installer.

File: src/winetricks [hunk 1]
```diff
@@ -13414,11 +13415,12 @@ load_vcrun2022()
             # Also install the 64-bit version
             # 2022/08/05: 14.32.31332 @ https://download.visualstudio.microsoft.com/download/pr/7331f052-6c2d-4890-8041-8058fee5fb0f/CE6593A1520591E7DEA2B93FD03116E3FC3B3821A0525322B0A430FAA6B3C0B4/VC_redist.x64.exe 8ae59d82845159db3a70763f5cb1571e45ebf6a1adfecc47574ba17b019483a0
             # 2023/04/30: 14.34.31938 @ https://download.visualstudio.microsoft.com/download/pr/8b92f460-7e03-4c75-a139-e264a770758d/26C2C72FBA6438F5E29AF8EBC4826A1E424581B3C446F8C735361F1DB7BEFF72/VC_redist.x64.exe 26c2c72fba6438f5e29af8ebc4826a1e424581b3c446f8c735361f1db7beff72
+            # 2023/07/04: 14.36.32532 @ https://download.visualstudio.microsoft.com/download/pr/eaab1f82-787d-4fd7-8c73-f782341a0c63/917C37D816488545B70AFFD77D6E486E4DD27E2ECE63F6BBAAF486B178B2B888/VC_redist.x64.exe
```
Comment (gverm) [inline src/winetricks:13418]: @Gcenx you forgot to add the new sha256 here too

**Review by gverm — CHANGES_REQUESTED** [review]

File: src/winetricks (hunk 1 above)
Comment (Gcenx) [reply src/winetricks]: @gverm fixed, just can’t rebase currently as I’m on mobile

That’s what happened when you do this shot before morning coffee

**Review by gverm — APPROVED** [review]

**Pull request merged**